source/_templates/layout.html

```diff
@@ -0,0 +1,31 @@
+{% extends "!layout.html" %}
+
+{% block extrahead %}
+    {# Get Roboto font from Google #}
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+
+    {% if pagename == "index" %}
```

Comment:
I'm not sure it is. if you specify a html file for `html_additional_pages`, it doesn't include the `index.rst` at all and I can't find a way to 'call' the rst from the html template. 

I guess it's a tradeoff between having a cleaner `layout.html` (including the index page content in `index.html` and getting rid of `index.rst`) or making it easier for anyone to edit the index page from the rst.

Reply:
So can we not include the jinja template syntax in the custom template too?

Reply:
Yes, so you can make `index.html` extend from `layout.html` so that jinja renders the navbar etc, but it wont render the `.rst` as well with `html_additional_pages`. 

Couldn't find out how to either specify to jinja to use the `.rst`, or specify in the `.rst` to use a different template?